### ISSUE TITLE
Scopes are overwritten

### ISSUE THREAD

**Issue by mattmatters** [simmatrix/vue-google-auth#12]
`config = Object.assign(options, { scope: 'profile email https://www.googleapis.com/auth/plus.login' })`

This overwrites any user provided scopes

Changing to `config = Object.assign({ scope: 'profile email https://www.googleapis.com/auth/plus.login' }, options)` would preserve expected behavior since no one can add scopes anyway, but allow the optional parameter.

**Comment by darior87** [comment]
did you find a solution for this?
I have the same problem. I want to add "google calendar" scope but as you wrote, it's not possible at this time.

**Comment by mattmatters** [comment]
Hey darior87!

I just switched the object assign around and it works great for me, I'm using the Google Analytics scope.

This [commit](https://github.com/mattmatters/vue-google-auth/commit/b1560984ac70f7b6529fa2c23eea98b4eb857aee) does it on my fork. Let me know if that works for you!

**Comment by darior87** [comment]
Yes, I cloned also this repo and created my own with this fix and it works. thanks to your comment!
